README.md

```diff
@@ -101,3 +101,18 @@ We are working on a [markdown version](https://github.com/glpi-project/doc)
  * [TinyMCE v4.7.1](https://github.com/tinymce/tinymce) LGPL2.1
  * [Underscore.js v1.8.3](https://github.com/jashkenas/underscore) MIT
  * [Lodash v unknown](https://github.com/lodash/lodash) MIT
+
+
+## Support
+GLPI is a living software. Improvements are continuously made, new functionality is being developed, and issues are being fixed.
+
+To ease support and development, we need your help when encountering issues.
+There is a GLPI version typical lifecycle:
+ * A new major version (9.3) is released.
+ * Minor versions (9.3.x), fixing bugs or issues, are published approximately every 3 months.
+   Please consider updating to the latest realeased minor version if you encounter some bugs or performance issues.
+ * 15 to 18 months after major version realesed, a new major version (9.4) is released
+   Both new and previous major versions are supported, with latest minor version update.
```

Comment:
This is wrong. There is only one major version supported. Old major is no longer supported when the new major has been released.

Reply:
I will upadte with this information :)